custom-recipes/wordcloud-recipe/recipe.json

```diff
@@ -0,0 +1,75 @@
+// This file is the descriptor for the Custom code recipe wordcloud-recipe
+{
+    // Meta data for display purposes
+    "meta": {
+        // label: name of the recipe as displayed, should be short
+        "label": "Wordcloud",
+        // description: longer string to help end users understand what this recipe does
+        "description": "Generate wordclouds based on a text column",
+        // icon: must be one of the FontAwesome 3.2.1 icons, complete list here at https://fontawesome.com/v3.2.1/icons/
+        "icon": "icon-cloud"
+    },
+    "kind": "PYTHON",
+    "selectableFromDataset": "input",
+    "inputRoles": [
+        {
+            "name": "input",
+            "label": "Input dataset",
+            "description": "Dataset on which to compute the wordclouds",
+            "arity": "UNARY",
+            "required": true,
+            "acceptsDataset": true,
+            "acceptsManagedFolder": false
+        }
+    ],
+    "outputRoles": [
+        {
+            "name": "output",
+            "label": "Output folder",
+            "description": "Folder to store the wordclouds images",
+            "arity": "UNARY",
+            "required": true,
+            "acceptsDataset": false,
+            "acceptsManagedFolder": true
+        }
+    ],
+    "paramsPythonSetup": "get_language_list.py",
+    "params": [
+        {
+            "name": "text_column",
+            "type": "COLUMN",
+            "columnRole": "input",
+            "label": "Text column",
+            "description": "Text column used to generate the wordclouds",
```

Comment:
```suggestion
```
I think we don't need to add descriptions if the parameter label is self-explanatory. Too much text in an interface is not necessarily useful.